### DISCUSSION TITLE
Random crash on song select screen

### DISCUSSION THREAD

**Discussion by DutchEllie** [ppy/osu#24894] [Q&A]
### Type

Crash to desktop

### Bug description

I am unsure how this crash happened, the game just kinda... went.

I was playing Blue Zenith and after I had completed the map, I went onto the song select screen. I hovered over a few of my older scores to compare the scores, and then the game just crashed. 

### Version

2023.914.0

### Logs

[input.log](https://github.com/ppy/osu/files/12706839/input.log)
[performance.log](https://github.com/ppy/osu/files/12706840/performance.log)
[database.log](https://github.com/ppy/osu/files/12706841/database.log)
[network.log](https://github.com/ppy/osu/files/12706842/network.log)
[runtime.log](https://github.com/ppy/osu/files/12706843/runtime.log)

**Comment by peppy** [comment]
Crash logs don't show an issue, which suggests it's something system level. Can you check your system logs?

**Comment by DutchEllie** [comment]
The only thing I could find was this from `dmesg`.

```
[Sat Sep 23 17:57:24 2023] Audio (GameThre[115681]: segfault at 8 ip 00007fb998405054 sp 00007fb9cd7f96f0 error 4 in libbassmix.so[7fb998400000+a000] likely on CPU 39 (core 20, socket 0)
[Sat Sep 23 17:57:24 2023] Code: af 53 34 48 29 d0 eb 7f 48 8b 49 10 48 85 c9 74 0a 48 8b 39 48 39 f7 7f ef eb 15 48 83 c8 ff 84 d2 75 64 48 8b 83 50 01 00 00 <48> 8b 40 08 eb 57 48 8b 41 08 48 8b 49 18 48 85 c9 74 4a 48 8b 51
```

**Reply by peppy** [reply]
Yeah, so something in audio went wrong.

Unless you can reproduce 100% with a scenario, I doubt we'll be able to work with this. Either an issue in bass, or more likely, linux.

**Comment by DutchEllie** [comment]
Well, I've been having issues with my audio on Linux these last few days now. I can't reproduce the issue, let alone reliably. Sorry for wasting your time, I guess.  ---- On Mon, 25 Sep 2023 06:24:16 +0200  ***@***.***  wrote ----
Yeah, so something in audio went wrong.
Unless you can reproduce 100% with a scenario, I doubt we'll be able to work with this. Either an issue in bass, or more likely, linux.

**Comment by sineplusx** [comment]
This also happens to me irregularly. Last crash was yesterday with a similar signature in the kernel logs.
```
[848.324517] Audio (GameThre[4025]: segfault at 20 ip 00007f0bbe1f00ca sp 00007f0adbc46600 error 4 in libbass.so[7f0bbe1c6000+38000]
[848.324524] Code: e9 75 13 48 85 d2 74 05 48 89 02 eb 11 48 89 83 90 00 00 00 eb 08 48 89 ca 48 85 c0 75 da 48 8b 83 90 00 00 00 8b 75 20 31 d2 <39> 70 20 7c 10 48 8b 08 48 89 c2 48 85 c9 74 0a 48 89 c8 eb eb 48
``` 
I believe this has only occured on song select for me. Pretty hard to troubleshoot since its a compiled library.

**Comment by LianSheng197** [comment]
It happened to me a couple days ago.
A similar error message can be found by looking at `/var/log/kern.log`.

![image](https://github.com/ppy/osu/assets/18535495/a420025a-69a5-45eb-ac03-350d5eb08d4d)
```
Oct  9 22:07:51 home2204 kernel: [4832652.017725] Audio (GameThre[52991]: segfault at 20 ip 00007f359422a0ca sp 00007f35457fb560 error 4 in libbass.so[7f3594200000+38000] likely on CPU 4 (core 0, socket 0)
```

Since I've had several unpredictable crashes with different versions of the game in the past, I remembered to save the entire log before restarting the game this time.

[logs_Oct09_2023.zip](https://github.com/ppy/osu/files/12872626/logs_Oct09_2023.zip)

Unlike the OP, I was playing when crashed in the middle of the beatmap.

----
```
    OS: Ubuntu 22.04.3 LTS x86_64
Kernel: 6.2.0-26-generic
   CPU: AMD Ryzen 5 3400G
```
